### PR TITLE
Add links to centos7-master and centos7-master-head

### PR DESCRIPTION
**Pull request by javierpena**
Trying to explain what each one means.

### CODE REVIEW

**Comment by hguemar** [comment]
:+1:

**Comment by yazug** [comment]
is this the rdoinfo pinned to upper constraints vs master for all difference?

**Comment by javierpena** [comment]
Yes, that's the only difference.

**Comment by yazug** [comment]
Something reflecting that not all version but a select set of libraries are pinned and more information on which ones can be extracted from rdoinfo.

**Comment by javierpena** [comment]
@yazug I've added some more detail on a footnote, let's see if it clarifies what is built.

**Comment by yazug** [comment]
+1

File: index.html [hunk 1]
```diff
@@ -161,6 +161,8 @@
                       </tr>
                     </tbody>
                   </table>
+                  <br><a name="footnote1"> [*] The newton-uc tag in <a href="https://github.com/redhat-openstack/rdoinfo/blob/master/rdo.yml">rdoinfo</a> defines the exact tags being packaged for each project. Not every package is built from a tagged release, only those present in the <a href="http://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt">upper-constraints</a> file, used by the upstream OpenStack CI. The rest of the packages are built from the master branch.
```
Comment (apevec) [inline index.html]: :-1: this is now ocata-uc

**Pull request closed without merging**